frontends/web/src/routes/account/account.tsx

```diff
@@ -113,12 +106,26 @@ class Account extends Component<Props, State> {
         if (this.props.code !== prevProps.code) {
             this.onStatusChanged();
             this.checkSDCards();
+            unsubscribe(this.subscribtions);
+            this.subscribe();
         }
         if (this.deviceIDs(this.props.devices).length !== this.deviceIDs(prevProps.devices).length) {
             this.checkSDCards();
         }
     }
 
+    private subscribe() {
+        this.subscribtions.push(
+            syncAddressesCount(this.props.code, (code, syncedAddressesCount) => {
+                if (code === this.props.code) {
+                    this.setState({ syncedAddressesCount });
+                }
+            }),
+            statusChanged(this.props.code, () => this.onStatusChanged()),
+            syncdone(this.props.code, () => this.onAccountChanged()),
```

Comment:
@x1ddos I had to change this https://github.com/thisconnect/bitbox-wallet-app/commit/bbd8e4dabb8a45c705724c2063f34dc122f36571


Switching accounts during account sync did not update the
subscriptions correctly.

Changed:

- use componentDidUpdate instead of componentWillReceiveProps
  (which is deprecated in react).
- the functions passed to syncdone and statusChanged were called
  with the wrong this, so the callbacks could not access
  this.props anymore. Passing an arrow function will call the
  function with the correct this.

Reply:
Right. That was the reason for re-creating the frontend-new-api branch. :+1: